apps/condo/domains/meter/access/CommunalMeter.js

```diff
@@ -0,0 +1,79 @@
+// @ts-nocheck
```

Comment:
?

Reply:
its codegen

Reply:
It doesn't mean, that you should not change it :)
Why do we need to multiply `ts-nocheck` amount in the project?